ios/src/TGMapViewController.h

```diff
@@ -22,13 +42,34 @@
 - (void)recognizer:(UIGestureRecognizer *)recognizer didRecognizeShoveGesture:(CGPoint)location;
 @end
 
+NS_ASSUME_NONNULL_END
+
 struct TileID;
 @interface TGMapViewController : GLKViewController <UIGestureRecognizerDelegate>
 
-@property (assign, nonatomic) Tangram::Map* map;
+@property (assign, nonatomic, nullable) Tangram::Map* map;
 @property (assign, nonatomic) BOOL continuous;
-@property (weak, nonatomic) id<TangramGestureRecognizerDelegate> gestureDelegate;
+@property (weak, nonatomic, nullable) id<TangramGestureRecognizerDelegate> gestureDelegate;
+
+// The following are computed property. They return sensible defaults when the above .map property is nil
+@property (assign, nonatomic) TangramCameraType cameraType;
+@property (assign, nonatomic) TangramGeoPoint position;
+@property (assign, nonatomic) float zoom;
+@property (assign, nonatomic) float rotation;
+@property (assign, nonatomic) float cameraTilt;
 
 - (void)renderOnce;
 
+- (void)setPositionEasedAtPosition:(TangramGeoPoint)position withDuration:(float)duration;
+
+- (void)setPositionEasedAtPosition:(TangramGeoPoint)position withDuration:(float)duration withEaseType:(TangramEaseType)easeType;
+
+- (void)setZoomEased:(float)zoomLevel withDuration:(float)duration;
+
+- (void)setZoomEased:(float)zoomLevel withDuration:(float)duration withEaseType:(TangramEaseType)easeType;
+
+- (void)setRotationEased:(float)rotation withDuration:(float)duration;
+
+- (void)setRotationEased:(float)rotation withDuration:(float)duration withEaseType:(TangramEaseType)easeType;
```

Comment:
Agreed, and updated.

As for cameraTilt, yep I missed exposing the eased variants. Good catch. Added.